concepts/booleans/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "Learn about boolean operations.",
```

Comment:
Concept blurbs should be more like descriptions, see https://github.com/exercism/csharp/blob/main/concepts/booleans/.meta/config.json#L2 for an example.

Reply:
I tried to improve it